containerd/main.go

```diff
@@ -55,6 +55,11 @@ var daemonFlags = []cli.Flag{
 		Value: &cli.StringSlice{},
 		Usage: "specify additional runtime args",
 	},
+	cli.IntFlag{
+		Name:  "pprof-port",
```

Comment:
Would it be better for this to be `pprof-address` so that you are not binding to all interfaces and could specify to bind this only to localhost?